proposals/lip-0022.md

```diff
@@ -145,12 +145,12 @@ As in the previous subsection, the random seed computation can be performed effi
 
 ### Round Computation
 
-A round is composed of 103 blocks now.
+A round is composed of 103 blocks now. In the case of having less than 103 delegates in the forging list, the forging list is looped again to assign the remaining slots (i. e., as it is done when delegates miss their forging slot).
```

Comment:
Maybe here, the information could be more general. Something like: "The block slots in the round are assigned to the delegates by looping over the delegate list." And if we want some additional info, maybe "This implies that a delegate may forge multiple blocks in a round if some delegates missed their forging slot, or if the forging list is shorter than the round length."

Reply:
I like Maxime's formulation too, just a minor thing, I'd say: "The block slots in the round are assigned to the delegates by looping over the FORGING list FROM THE START. This implies that a delegate may forge multiple blocks in a round if some delegates missed their forging slot, or if the forging list is shorter than the round length."

Reply:
I like the suggestion by @ricott1 as I think it is more clear than the current one.

Reply:
thanks for the suggestion, updated as @ricott1  said.